packages/react-native-editor/CHANGELOG.md

```diff
@@ -11,6 +11,16 @@ For each user feature we should also add a importance categorization label  to i
 
 ## Unreleased
 
+## 1.73.1
+
+-   [*] [Space block] Fix crash when changing the height value using the text input [#40053]
```

Comment:
I just noticed a small typo here ("Space" should be "Spacer"). I'll address this as part of future commits, to avoid slowing down the release process.